firefox-ios/WidgetKit/ImageButtonWithLabel.swift

```diff
@@ -90,19 +90,19 @@ struct ImageButtonWithLabel: View {
                         }
                         Spacer()
                         if link == .search && isSmall {
-                            Image("searchLarge")
+                            Image(decorative: "searchLarge")
```

Comment:
Just found out about this Image [init](https://developer.apple.com/documentation/swiftui/image/init(decorative:bundle:)). Does it mean the VoiceOver will skip it?

Reply:
@razvanlitianu Yes, decorative images will be skipped in VoiceOver. Please let me know if any of the images in this PR should not be skipped and should have an a11y label instead.